packages/popover/src/main.vue

```diff
@@ -186,6 +194,12 @@ export default {
     handleAfterLeave() {
       this.$emit('after-leave');
       this.doDestroy();
+    },
+    cleanup: function cleanup() {
+      if (this.openDelay) {
+        clearTimeout(this._timer);
+        this.showPopper = false;
```

Comment:
Popover component uses `vue-popper` mixin which contains `beforeDestroy` and `deactivated` hooks. In this function, there's no need to reset `showPopper`.

https://github.com/ElemeFE/element/blob/0e7a7d345f508531cfb170a613a6ece1a2c398b7/src/utils/vue-popper.js#L188-L199